.pre-commit-config.yaml

```diff
@@ -76,7 +76,7 @@ repos:
       - id: shfmt
         args: ["-i", "4"]
       - id: shellcheck
-        files: deploy/azure/.*sh
+        files: (deploy/azure/.*sh|scripts/publish_cft.sh)
```

Comment:
Not for this PR, but we should probably apply it on all `*.sh` files right?

Reply:
I agree, https://github.com/elastic/cloudbeat/issues/1364